src/super_gradients/training/models/detection_models/pp_yolo_e/pp_yolo_e.py

```diff
@@ -59,12 +64,14 @@ def set_dataset_processing_params(
         self._default_nms_iou = iou or self._default_nms_iou
         self._default_nms_conf = conf or self._default_nms_conf
 
-    def _get_pipeline(self, iou: Optional[float] = None, conf: Optional[float] = None) -> DetectionPipeline:
+    @lru_cache(maxsize=1)
```

Comment:
I added a cache of 1, to keep in memory the latest pipeline used:
- If the user uses the same `iou`/`conf`, we keep the pipeline as is
    - to not instantiate everything again
    - to avoid fusing the model every time
- If the user sets different iou/conf, we instantiate a new pipeline, which will take a little more time

Reply:
In practice they do, either working with a lot of images that dont fit into memory, processing a big video, or doing custom stuff with webcam stream